hc_cluster_test/conductor_handle.ts

```diff
@@ -0,0 +1,93 @@
+import { dnaInterfaceId } from './config'
+import { connect } from '@holochain/hc-web-client'
+import * as kill from 'tree-kill'
+
+type ChildProcess = any
+type WebsocketConnection = any
+
+type Call = (...segments: Array<string>) => (params: any) => Promise<any>
```

Comment:
@willemolding I just borrowed the type defs for these functions from hc-web-client, but is there some way that I could import or reference them instead?